### ISSUE TITLE
Allow filtering by MP / MLA Constituencies on Story Dashboard 

### ISSUE THREAD

**Issue by batpad** [klpdotorg/dubdubdub#525]
Nice to have.

**Comment by meghaklp** [comment]
Tracking this as well

**Comment by batpad** [comment]
Need to add this to the select2 list (easy), and allow this filtering on the API.

Assigning to @harisibrahimkv right now - to add it as a filter option in the API. @harisibrahimkv we can discuss this, should be fairly straightforward.

**Comment by harisibrahimkv** [comment]
@batpad What are MP / MLA Constituencies in the db/models?

**Comment by batpad** [comment]
@harisibrahimkv ah sorry for the late response here -- models are here: https://github.com/klpdotorg/dubdubdub/blob/develop/apps/schools/models/elected_reps.py#L35

My guess is the filter is something like school__schoolelectedrep__mla_const and school__schoolelectedrep__mp_const .. do you want to give it a shot?

Thanks!

**Comment by harisibrahimkv** [comment]
@batpad Yeah, I am giving it a shot. I am assuming this filter is applicable for all three Volumes, Detail and Summary endpoints. However, will you be passing in the model_id or one of the following in order to filter for the mp or mla constituency?

    elec_comm_code = models.IntegerField(blank=True, null=True)
    const_ward_name = models.CharField(max_length=300, blank=True)

**Comment by harisibrahimkv** [comment]
The backend is done. @meghaklp @batpad Take it away!

**Comment by meghaklp** [comment]
What's to be done on front end? Constituencies show up in search box with the auto completion, no? @batpad

**Comment by harisibrahimkv** [comment]
@meghaklp @batpad Oh okay. Do test it out and if the frontend is done, feel free to close the issue.

**Comment by meghaklp** [comment]
Looks like the search box expects some kind of json/geojson when this request is made - "/api/v1/search" 

this is the response right now and doesn't have mp mla boundaries

```
"properties": {"id": 8777, "name": "bangalore central", "type": "project", "school_type": "preschool"}}, {"geometry": {"type": "Point", "coordinates": [77.594376, 12.971606]}, "type": "Feature", "properties": {"id": 8778, "name": "bangalore state", "type": "project", "school_type": "preschool"}}, {"geometry": {"type": "Point", "coordinates": [77.6633084, 12.8420391]}, "type": "Feature", "properties": {"id": 8779, "name": "bangalore south(banashankari)", "type": "project", "school_type": "preschool"}}]}, "pre_schools": {"type": "FeatureCollection", "features": []}}
```

**Comment by harisibrahimkv** [comment]
@meghaklp Has this been done?
